src/commands/build/mod.rs

```diff
@@ -1,17 +1,29 @@
+// FIXME: move
```

Comment:
It's not clear what needs to happen here.

Reply:
We can move the krate and package code into build because it's captured by WorkerBundle.